Packs/Spamcop/Integrations/Spamcop/Spamcop.yml

```diff
@@ -0,0 +1,31 @@
+commonfields:
+  id: Spamcop
+  version: -1
+name: Spamcop
+display: Spamcop
+category: Data Enrichment & Threat Intelligence
+description: SpamCop is an email spam reporting service, integration allow checking
+  the reputation of an IP address
+configuration: []
+script:
+  script: ''
+  type: python
+  commands:
+  - name: ip
+    arguments:
+    - name: ip
+      required: true
+      description: The IP which details you want to find
+      default: true
+      isArray: true
+    outputs:
+    - contextPath: Spamcop.IP
+      description: IP details
+      type: String
```

Comment:
Please add here also the dbotscore paths + IP paths, for example:
DBotScore.Indicator, IP.Address, etc..
You can find all relevant keys and descriptions here And also add these as command outputs. You can find the standards here: https://xsoar.pan.dev/docs/integrations/context-standards

Reply:
done